libraries/splunk_template.rb

```diff
@@ -60,6 +61,10 @@ def variables(args = nil)
         end
       end
 
+      def sensitive(args = nil)
```

Comment:
Should our monkey patch just be this? Maybe with an additional `default: false` validator? This resource should then be able to use it passively without duplicating code (or overriding code in future chef versions).